### ISSUE TITLE
Inconsistent naming gpkg_contents(identifier) / gpkg_data_columns (name)

### ISSUE THREAD

**Issue by cclark1984** [opengeospatial/geopackage#229]
In gpkg_contents the identifier column is described as, "A human-readable identifier (e.g. short name) for the table_name content."

In the gpkg_data_columns the name column is described as, "A human-readable identifier (e.g. short name) for the column_name content."

I believe that the two columns should be consistently named "identifier" or "name", preferably "identifier".

**Comment by cclark1984** [comment]
On second thought, do we even require these fields at all? Neither are primary keys and although they are unique, they are also nullable, therefore they can't be used reliably as an identifier.

**Comment by jyutzler** [comment]
Well, I don't know that we require them but I think applications out there are expecting them to be there. I am reluctant to remove something like this. It is unfortunate, but I don't see how we can endorse this change. What is the cost-benefit story?

**Comment by cclark1984** [comment]
I agree that we can't rename or remove this column without breaking compatibility with existing geopackages, however I don't think that is a valid reason for not improving the specification. We should at least keep a document of changes we would like to investigate for a next version specification, once we have collected enough suggestions, we can update the spec with multiple improvements that warrant the cost of implementers having to make changes.

**Comment by restjohn** [comment]
> I agree that we can't rename or remove this column without breaking compatibility with existing geopackages, however I don't think that is a valid reason for not improving the specification.

I agree whole-heartedly with that.  I think there is great value in cleaning up technical debt to make the spec more clear and intuitive.  That would encourage better adoption and interoperability.  I believe GeoPackage is still young enough to make changes like that.

Identifier is technically the long form of the abbreviation 'ID', which is universally inferred as a uniquely identifying attribute.  I think better choices to convey human-readable text would be title, label, or name, in order of my personal preference.  I personally tend to ascribe more formal/strict constraints to the text a 'name' attribute can contain, such as a variable name in a programming language, a database column name, or an XML [NCName](https://www.w3.org/TR/xml-names/#NT-NCName).

**Comment by jerstlouis** [comment]
SWG 2026-08-18: We will likely tackle this issue in two steps:

1. We will clarify the standard to explain that for historical reason the columns are named differently. This will be addressed in a minor revision 1.5
2. Providing we eventually work on a major revision 2.0, we will rename the columns consistently. For example, we could rename `name` to `title`, and get rid of `title` in `gpkg_data_columns`, and rename `identifier` to `title` in `gpkg_contents`. This would be consistent with the OGC API JSON schemas, for example.

**Comment by heidivanparys** [comment]
A small test with gdal and QGIS shows that they fetch the human-readable column names from ` name`, not from `title`.

<img width="980" height="343" alt="Image" src="https://github.com/user-attachments/assets/be797e59-239b-498d-a5e8-1c696b6169c7" />

<img width="1345" height="190" alt="Image" src="https://github.com/user-attachments/assets/c2afe1da-4e37-4658-9a25-d2a5609ca1b7" />

```
Layer name: namedplace
Metadata:
  IDENTIFIER=named place
  DESCRIPTION=Any real world entity referred to by one or several proper nouns.
Geometry: Point
Feature Count: 0
Layer SRS WKT:
[...]
Data axis to CRS axis mapping: 1,2
FID Column = objectid
Geometry Column = geometry
nametext: String (0.0) NOT NULL, alternative name="text (name)", comment=Way the name is written.
namegrammaticalgender: String (0.0) NOT NULL, alternative name="grammatical gender (name)", domain name=grammaticalgendercode, comment=Class of nouns reflected in the behaviour of associated words.
```
